packages/vite/src/build.ts

```diff
@@ -13,15 +13,23 @@ export function emberBuild(command: string, mode: string, resolvableExtensions:
 
   if (command === 'build') {
     return new Promise((resolve, reject) => {
-      const child = fork('./node_modules/ember-cli/bin/ember', ['build', '--environment', mode], { env });
+      const child = fork(
+        './node_modules/ember-cli/bin/ember',
+        ['build', '--environment', mode, '-o', 'tmp/compat-prebuild', '--suppress-sizes'],
```

Comment:
`--suppress-sizes` is unrelated, but fixes getting the following red error message on production builds:
```
No asset files found in the path provided: dist/
```